File: Programmers/Hash/완주하지 못한 선수.py
def solution(participant, completion):
    answer = ''
    participant.sort()
    completion.sort()
    for i in range(len(participant)):
        try:
            if participant[i]!=completion[i]:
                answer = participant[i]
                break
        except:
            answer = participant[i]
            break
    return answer

File: Programmers/Hash/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    sort(participant.begin(), participant.end());
    sort(completion.begin(),completion.end());        
    int n = completion.size();
    for(int i =0; i<n; i++)
        if(participant[i]!=completion[i])
            return participant[i];
    return participant[n];
}